src/utils/githubApi.ts

```diff
@@ -60,12 +45,14 @@ export class GitHubRemote {
   /**
    * Returns an HTTP-based git remote with embedded HTTP basic auth
    *
+   * Using dummy username as it does not matter for cloning
+   *
    * It MAY contain sensitive information (e.g. API tokens)
    */
   public getRemoteStringWithAuth(): string {
     const authData =
-      this.username && this.apiToken
-        ? `${this.username}:${this.apiToken}@`
+      this.apiToken
+        ? `dummyusername:${this.apiToken}@`
```

Comment:
placeholderusername is more appropriate from an ableist-language perspective

Reply:
Updated!